test/e2e/traceflow_test.go

```diff
@@ -36,10 +36,9 @@ type testcase struct {
 	expectedResults []v1alpha1.NodeResult
 }
 
-// TestTraceflow verifies if traceflow can trace intra/inter nodes traffic with some NetworkPolicies set.
-func TestTraceflow(t *testing.T) {
-	skipIfProviderIs(t, "kind", "Inter nodes test needs Geneve tunnel")
-	skipIfNumNodesLessThan(t, 2)
+// TestTraceflow_IntraNode verifies if traceflow can trace intra node traffic with some NetworkPolicies set.
+func TestTraceflow_IntraNode(t *testing.T) {
```

Comment:
nit: s/TestTraceflow_IntraNode/TestTraceflowIntraNode

I noticed we don't usually use underscores for function names / test case names

Reply:
fixed